kasenvr/project-athena
# Re-branding of help dialogs and tablet login

Fixed branding on Help dialog images and Tablet login

To test:
in VR or Desktop
Open help-Controls Reference check each window says support@projectathena.io near the bottom

In VR  logout then log in. Check that there is no Highfidelity branding or links.
